src/component/calculator.js

```diff
@@ -1,40 +1,67 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 /* eslint-disable react/prop-types */
```

Comment:
Kindly remove this disabled linter, linting improves readability, removes errors. 👍 

Please check for other occurrences 🙂